examples/helm-deployment/skaffold.yaml

```diff
@@ -3,7 +3,7 @@ kind: Config
 build:
   artifacts:
   - image: skaffold-helm
-manifests:
+deploy:
```

Comment:
Is this required as part of this change? Is this for legacy helm deployer? If that is the case, should we add another example.

Reply:
This is not required for the fix, but we should fix this. This actually uses helm render and kubectl deployer, not as the name suggested, this also was the cause that I didn't catch this issue when fixing cleanup/ delete issues. 

Added another example